docker-compose.yaml

```diff
@@ -16,7 +16,8 @@ services:
       - db
     environment:
       envname: 'dkrcompose'
-      AWS_DEFAULT_REGION: "eu-west-1"
+      AWS_REGION: "${AWS_REGION}"
```

Comment:
We could consider specify the default values if the env variables are not specify. e.g.:
```
AWS_REGION: "${AWS_REGION:-eu-west-1}"
AWS_DEFAULT_REGION: "${AWS_DEFAULT_REGION:-eu-west-1}"
```
Otherwise, we have to provide them all the time from the console